homeassistant/generated/integrations.json

```diff
@@ -1487,6 +1487,12 @@
       "config_flow": true,
       "iot_class": "cloud_polling"
     },
+    "elevenlabstts": {
+      "name": "ElevenLabs Text-to-Speech (TTS)",
+      "integration_type": "hub",
+      "config_flow": true,
```

Comment:
```suggestion
      "config_flow": true,
      "integration_type": "service",
```

Reply:
I won't add the suggestion but generate the file locally instead after the other comments have been resolved

Reply:
Aight, done now!